docs/stoploss.md

```diff
@@ -27,55 +21,101 @@ So this parameter will tell the bot how often it should update the stoploss orde
 This same logic will reapply a stoploss order on the exchange should you cancel it accidentally.
 
 !!! Note
-    Stoploss on exchange is only supported for Binance (stop-loss-limit), Kraken (stop-loss-market) and FTX (stop limit and stop-market) as of now.
+    Stoploss on exchange is only supported for Binance (stop-loss-limit), Kraken (stop-loss-market) and FTX (stop limit and stop-market) as of now.  
+    Do not set too low stoploss value if using stop loss on exhange!
```

Comment:
I think it should be explained why it's not adviced to use a thight/low stoploss value if using stoploss on exchange. If the user understands how and what, it's easier to actually apply it to your strategy.

Reply:
Did a try on explaining it.. please have a look